server/py/framework/db/sqldb/db.py

```diff
@@ -4807,6 +4816,9 @@ def _find_functions(
         if hash_key is not None:
             query = query.filter(Function.uid == hash_key)
 
+        if kind is not None:
```

Comment:
if kind is an empty string?

Reply:
there are function with kind empty (used to be local or handler, cant really recall) - and that should take action